src/Parser/Shaman/Enhancement/Modules/Items/Tier21_2set.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+
+class Tier21_2set extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  extraDmg = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasBuff(SPELLS.ENHANCE_SHAMAN_T21_2SET_EQUIP.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (this.combatants.selected.hasBuff(254308)) {  //ForceOfTheMountain.id
```

Comment:
Use SPELLS.FORCE_OF_THE_MOUNTAIN.id ?

Reply:
yup that's an oversight from debugging thx